public/css/dashboard-resources.css

```diff
@@ -0,0 +1,51 @@
+.Resources {
+	margin: 100px auto 34px;
+}
+.row {
+	margin-bottom: 20px;
+}
+/* for reference links */
+.row a {
+	color: #e1489a;
+}
+
+.row p {
+	margin-top: 15px;
+	font-size: 14px;
+	font-style: normal;
+	font-weight: normal;
+	color: black;

```

Comment:
would be ideal to consistently use hex for all colors used, if possible

Reply:
changed